internal/controller/biossettings_controller_test.go

```diff
@@ -21,6 +21,11 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// added const field for go linting
+const (
+	testBarValue = "bar"
+)
+
```

Comment:
Probably, we should not run the `goconst` (?) linter on the test files, instead of introducing an inconsistently used constant.